test_programs/cl2/3.lox

```diff
@@ -0,0 +1,19 @@
+---
+expected_error_type: none
+---
+// Instantiating classes in a function
+class Wizard {}
+class Dragon {}
+
+fun createCharacters() {
+  var merlin = Wizard();
+  var smaug = Dragon();
+  print "Characters created in fantasy world:";
+  print merlin;
```

Comment:
Either this (test 3) or test2 would be good place to test truthiness of instances - currently it is tested in test 1